packages/extension-base/src/koni/api/nft/transfer.ts

```diff
@@ -44,19 +44,6 @@ export function rmrkGetExtrinsic (substrateApi: _SubstrateApi, senderAddress: st
 }
 
 export function uniqueGetExtrinsic (substrateApi: _SubstrateApi, senderAddress: string, recipientAddress: string, params: Record<string, any>) {
-  try {
-    const itemId = params.itemId as number;
-    const collectionId = params.collectionId as number;
-
-    return substrateApi.api.tx.nft.transfer({ Substrate: recipientAddress }, collectionId, itemId, 1);
```

Comment:
Check back with Unique team to confirm the extrinsic

Reply:
Unique team confirmed